crates/router/src/db/kafka_store.rs

```diff
@@ -2131,3 +2133,27 @@ impl AuthorizationInterface for KafkaStore {
             .await
     }
 }
+
+#[async_trait::async_trait]
+impl HealthCheckInterface for KafkaStore {
```

Comment:
can we include kafka into the deep health check?

cc: @lsampras @ShivanshMathurJuspay